src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java

```diff
@@ -28,26 +33,40 @@
 public class BuildingLumberjack extends AbstractBuildingWorker
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2160)